pkg/imageserver/webdav.go

```diff
@@ -13,31 +13,30 @@ import (
 )
 
 const (
-	// CHROOT_SERVE_PATH is the path to server if we are performing a chroot
+	// chrootServePath is the path to server if we are performing a chroot
 	// this probably does not belong here.
-	CHROOT_SERVE_PATH = "/"
-	// AUTH_TOKEN_HEADER is the custom HTTP Header used
+	chrootServePath = "/"
+	// authTokenHeader is the custom HTTP Header used
 	// to authenticate to image inspector.
-	// We use a custom auth header instead of Authorization
+	// Clients must use a custom auth header
+	// instead of standard Authorization
 	// because Kubernetes Proxy strips the default Auth Header
 	// from requests
-	AUTH_TOKEN_HEADER = "X-Auth-Token"
+	authTokenHeader = "X-Auth-Token"
 )
 
 // webdavImageServer implements ImageServer.
 type webdavImageServer struct {
-	opts   ImageServerOptions
-	chroot bool
+	opts ImageServerOptions
 }
 
 // ensures this always implements the interface or fail compilation.
 var _ ImageServer = &webdavImageServer{}
 
 // NewWebdavImageServer creates a new webdav image server.
-func NewWebdavImageServer(opts ImageServerOptions, chroot bool) ImageServer {
+func NewWebdavImageServer(opts ImageServerOptions) *webdavImageServer {
```

Comment:
To make the return object testable (since the type is private, it's not possible to type cast this from the test package). Although that change should probably be part of #65 not this PR.

should this function use the interface type as its return value? no big deal if so; tests will just have to be in the same package (rather than a `*_test` package) so they'll be able to type cast